browser/ui/views/omnibox/brave_omnibox_view_views.cc

```diff
@@ -46,12 +56,15 @@ bool BraveOmniboxViewViews::AcceleratorPressed(
           : ui::ET_KEY_RELEASED,
       accelerator.key_code(), accelerator.modifiers());
   auto command = GetCommandForKeyEvent(event);
-
+  auto url_to_copy = GetURLToCopy();
   if ((GetTextInputType() != ui::TEXT_INPUT_TYPE_PASSWORD) &&
-      (command != ui::TextEditCommand::COPY || !SelectedTextIsURL())) {
+      (command != ui::TextEditCommand::COPY)) {
+    return OmniboxViewViews::AcceleratorPressed(accelerator);
+  }
+  if (!url_to_copy.has_value()) {
     return OmniboxViewViews::AcceleratorPressed(accelerator);
   }
-  ExecuteCommand(IDC_COPY_CLEAN_LINK, 0);
+  brave::CopySanitizedURL(chrome::FindLastActive(), url_to_copy.value());
```

Comment:
According to the parent implementations,
There is a pattern like below..
```
  OnBeforeUserAction();
   Do action like copy
  OnAfterUserAction();
```
We don't need it here?

Reply:
added, but Im not sure we need it actually